third_party/2and3/mypy_extensions.pyi

```diff
@@ -1,8 +1,21 @@
-from typing import Dict, Type, TypeVar, Union
+from typing import Dict, Type, TypeVar, Optional, Union
 
 _T = TypeVar('_T')
 
```

Comment:
Remove second empty line (in typeshed we tend to use a single empty line between different sorts of things).